multimodal/src/autogluon/multimodal/data/process_label.py

```diff
@@ -70,6 +70,7 @@ def __call__(
         labels: Dict[str, Union[int, float]],
         feature_modalities: Dict[str, Union[int, float, list]],
         is_training: bool,
+        load_only: bool = False,
```

Comment:
Does `mmdet_image` use label processor? I thought it uses only the mmdet processor.

Reply:
done 2 fixes we discussed offline and added a TODO here